client/src/components/Nav.js

```diff
@@ -115,6 +118,12 @@ class Nav extends Component {
 						<Link to={"/admin/mergePeople"}><NavItem>Merge people</NavItem></Link>
 					</SubNav>
 				}
+				
+				{externalDocumentationUrl &&
+					<li role="presentation">
+						<a href={externalDocumentationUrl} target="_blank">External documentation</a>
```

Comment:
I would used a named tab, e.g. `target="_extdocs"` instead of blank, so it always opens in the same tab instead of a new one each time.